src/commands/appSettings/AzureWebJobsStoragePromptStep.ts

```diff
@@ -45,7 +45,9 @@ export class AzureWebJobsStoragePromptStep<T extends IAzureWebJobsStorageWizardC
     }
 
     public shouldPrompt(context: T): boolean {
-        return !context.azureWebJobsStorageType && context.storageAccount === undefined && context.newStorageAccountName === undefined;
+        //Only want to prompt if no storage account it selected
+        context.azureWebJobsStorageType = (context.storageAccount !== undefined || context.newStorageAccountName !== undefined) ? 'azure' : undefined;
```

Comment:
You can use `!!` as a shorthand in JS to check to see if a value is truthy or not. Wasn't sure if you were familiar with it. It's a little goofy, but it's using `!` to first get the opposite boolean value, and then the next `!` converts it back. The one caveat is if 0 or "" would be valid inputs, this would give you the incorrect value.

```suggestion
        context.azureWebJobsStorageType = (!!context.storageAccount || !!context.newStorageAccountName) ? 'azure' : undefined;
```